### PR TITLE
feat(icons): new icons for PCH LS-3568

### PR DESCRIPTION
**Pull request by andrii-matviichuk**
## Jira

[[DS] Add PCH icons to design-system](https://littlespoon.atlassian.net/browse/LS-3568)

## Test Plan

Desk check to make sure all icon props were added and they are looking as expected.

## Affected Areas

Added new icons with stories and tests:
account ✅
add ✅ (was added previously with the name “Open”)
bill ✅
box ✅
calendar ✅
caret ✅
cart ✅
close ✅ (was added previously)
gift-card ✅
hamburger-menu ✅
logout ✅
subtract ✅(was added previously)

## Risk Analysis
Low risk, but we need to make sure that icons have proper and all the needed props

### CODE REVIEW

File: packages/icons/__tests__/BoxIcon.test.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+describe('with props.fill', () => {
+  it('renders box icon with props.fill', () => {
+    render(<BoxIcon fill="#bada55" />)
+    expect(screen.getByTitle('Box icon')).toBeInTheDocument()
+  })
+})
+
+describe('with props.notificationColor', () => {
+  it('renders box icon with props.notificationColor', () => {
+    render(<BoxIcon notificationColor="#bada55" />)
```
Comment (remarkablemark) [inline packages/icons/__tests__/BoxIcon.test.tsx:41]: Question: did this prop name come from ZeroHeight?

Comment (andrii-matviichuk) [reply packages/icons/__tests__/BoxIcon.test.tsx:41]: Nope, Here is it - https://zeroheight.com/3ddd0f892/p/699b01-icons/b/315972 . Should it be like a boolean with the name `notification`?

File: packages/icons/src/HamburgerMenuIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      size={size}
+      {...other}
+    >
+      {fill && <circle cx="12" cy="12" r="12" fill={fill} />}
+      <title>HamburgerMenu icon</title>
+      <desc>HamburgerMenu icon</desc>
```
Comment (remarkablemark) [inline packages/icons/src/HamburgerMenuIcon.tsx:40]:
```suggestion
      <title>Hamburger menu icon</title>
      <desc>Hamburger menu icon</desc>
```

**Review by remarkablemark — CHANGES_REQUESTED** [review]
Looks good overall, left comments

File: packages/icons/__tests__/BoxIcon.test.tsx (hunk 1 above)
Comment (remarkablemark) [reply packages/icons/__tests__/BoxIcon.test.tsx:41]: I think it works. I was wondering if there was a more semantic prop name but after seeing Storybook, I was able to understand what this is

@fjallkraken thoughts?

**Review by fjallkraken — APPROVED** [review]

File: packages/icons/__tests__/BoxIcon.test.tsx (hunk 1 above)
Comment (fjallkraken) [reply packages/icons/__tests__/BoxIcon.test.tsx:41]: There might be a better naming but I think it's clear enough if you have seen the icon

**Review by remarkablemark — COMMENTED** [review]
Let's denote the commit where `OpenIcon` was renamed to `AddIcon` as a [breaking change](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer)

**Pull request merged**